src/citation/constants.py

```diff
@@ -156,6 +156,7 @@
         "issued",
         "note",
         "journal-name",
+        "pdf_url",
```

Comment:
Lets change this to `"URL"` because `"pdf_url"` is not CSL compliant.

Reply:
we already have another field "URL" -- what should we do about that other field?

Reply:
The CSL fields are slightly limiting, I'm thinking the best way we can handle it is if we can transform our own "fields" JSON into CSL fields with a function, that way we can discard fields when we need a citation but can keep other metadata.

Reply:
CSL has the additional `"custom"` field, where we can enter our own key-value pairs

Reply:
ah, we should add it in there then. Is "custom" a JSON field?

Reply:
Yep.
In `schema.py`
```
            "custom": {
                "title": "Custom key-value pairs.",
                "type": "object",
                "description": "Used to store additional information that does not have a designated CSL JSON field. The custom field is preferred over the note field for storing custom data, particularly for storing key-value pairs, as the note field is used for user annotations in annotated bibliography styles.",
                "examples": [
                    {"short_id": "xyz", "other-ids": ["alternative-id"]},
                    {"metadata-double-checked": True},
                ],
            },
```

Reply:
Okay, made the change here. Will review again and deploy tomorrow